egs/swbd/s5c/local/nnet3/multi_condition/copy_ali_dir.sh

```diff
@@ -0,0 +1,78 @@
+#!/bin/bash
+
+# Copyright 2014  Johns Hopkins University (author: Vijayaditya Peddinti)
```

Comment:
I don't think it will be necessary to add this script once you make those other changes.
I'll talk to David separately to see if we can come up with a roadmap to make it unnecessary to have alignments, for purposes of training the speaker-id systems... this is very inconvenient.  I don't want to have a situation where the need for these alignments is making the scripts super complicated [see my other, long, comment].